src/shared/views/Login.tsx

```diff
@@ -2,20 +2,24 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import LoginBox, { Realm } from '../components/Login';
 import { AuthState } from '../store/reducers/auth';
-import { RootState } from '../store/reducers';
-import { StaticRouterProps } from 'react-router';
+import { StaticRouterProps, Redirect } from 'react-router';
+import { push } from 'connected-react-router';
 
 export interface LoginViewProps {
   authorizationEndpoint: string;
+  realms: Realm[];
   clientId: string;
   hostName: string;
   redirectUrl: string;
+  redirect(): void;
 }
 
 const Login: React.FunctionComponent<LoginViewProps> = props => {
-  const realms: Realm[] = [
-    { name: 'BBP', authorizationEndpoint: props.authorizationEndpoint },
-  ];
+  const { realms, redirect } = props;
+  if (realms.length === 0 || !realms) {
```

Comment:
the second test statement would also work in the first case, can be simplified to `!realms`

Reply:
fixed in 7172506e898cf880598ca2dfa8d90588445f222f